test/plugins/test_flake8_lint.py

```diff
@@ -126,7 +184,7 @@ def test_flake8_multiline(workspace):
         flake8_lint.pylsp_lint(workspace, doc)
 
     call_args = popen_mock.call_args[0][0]
-    assert call_args == ["flake8", "-", "--exclude=blah/,file_2.py"]
+    assert call_args == ["flake8", "-", "--exclude=blah/,file_2.py", "--stdin-display-name", "blah/__init__.py"]
```

Comment:
```suggestion
    init_file = "blah\\__init__.py" if os.name == "nt" else "blah/__init__.py"
    assert call_args == ["flake8", "-", "--exclude=blah/,file_2.py", "--stdin-display-name", init_file]
```

Reply:
oops, I've replaced it with os.path.join, which should also work. Please take another look :)

Reply:
No prob, that's fine too.